### PR TITLE
Remove KerasLayerWrapper for expandDim

### PR DESCRIPTION
**Pull request by Deliadd**

### CODE REVIEW

**Comment by Deliadd** [comment]
Refer to issue https://github.com/intel-analytics/zoo/issues/405

File: pyzoo/zoo/pipeline/api/keras/layers/core.py [hunk 1]
```diff
@@ -518,3 +518,19 @@ def __init__(self, dim, num_input_dims=INTMIN, return_value=True, input_shape=No
...
+
+class ExpandDim(ZooKerasLayer):
+    """
+    Expand_dim is an improved layer to suuport 1D input.
+    For example, if we get an 1D input with shape(3),
+    we will return the shape(1, 3) after we use expand_dim(0, input).
+    # Arguments
+    dim: The specified axis to expand dimension on.
+    input_shape: A shape tuple, not including batch.
+    """
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/keras/layers/core.py]: add a doc test here.

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/ExpandDimSpec.scala [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.zoo.pipeline.api.autograd.Parameter
+import com.intel.analytics.zoo.pipeline.api.keras.models.{Model, Sequential}
+import com.intel.analytics.zoo.pipeline.api.keras.serializer.ModuleSerializationTest
+
+import scala.util.Random
+
+class Expand_dimSpec extends KerasBaseSpec {
+  //  seq.add(new Expand_dim[Float](dim = 0))
```
Comment (zhichao-li) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/ExpandDimSpec.scala]: remove

**Comment by Deliadd** [comment]
Jenkins passed. ID is 2113.
Check the link here: http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Validation/2113/

**Pull request merged**